test/Lokad.ILPack.Tests/Lokad.ILPack.Tests.csproj

```diff
@@ -11,8 +11,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="3.0.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="CS-Script" Version="3.29.0" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
I'm not sure why an unit test project references `System.Net.Http`.

Reply:
I had introduced it because of a compilation error I had, but seems that this is not necessary. I'm removing this.

Reply:
OOps, my bad.

This is necessary in order to compile this:  
 [AssemblyGeneratorTest.TestInlineConstructorReference](https://github.com/arlm/ILPack/blob/25aa7d18aa5feed5e1fa688b9e7e2d9b57ba109a/test/Lokad.ILPack.Tests/AssemblyGeneratorTest.cs#L261-L300).


I think that should be kept.

Reply:
Agreed, this should be kept.